infrastructure/zk/src/server.ts

```diff
@@ -46,20 +45,7 @@ export async function externalNode(reinit: boolean = false, args: string[]) {
 
 async function create_genesis(cmd: string) {
     await utils.confirmAction();
-    await utils.spawn(`${cmd} | tee genesis.log`);
-
-    const date = new Date();
```

Comment:
Should we not keep this?

Reply:
There was a discussion where it seems that we're not using these logs anymore

Reply:
@mm-zk is it fine if we remove these log from here?

Reply:
AFAIK yes. @Deniallugo - FYI.

Reply:
i don't think we need this function at all